parsons/google/google_bigquery.py

```diff
@@ -371,6 +374,12 @@ def copy_from_gcs(
                 on the BigQuery client. The function will create its own if not provided. Note
                 if there are any conflicts between the job_config and other parameters, the
                 job_config values are preferred.
+            force_unzip_blobs: bool
+                If True, target blobs will be unzipped before being loaded to BigQuery.
+            compression_type: str
```

Comment:
This is only relevant for the decompression and not the job config? I wonder if we make that clear by calling it "force_unzip_compression_type" and "force_unzip_new_file_extension